config/nav.yml

```diff
@@ -209,7 +209,14 @@ nav:
       # Eventing - troubleshooting
       - Troubleshooting:
         - Debugging: eventing/troubleshooting/README.md
-      - Experimental Features: eventing/experimental-features.md
+      - Experimental Features:
+        - About Eventing experimental features: eventing/experimental-features/README.md
+        - DeliverySpec.Timeout field: eventing/experimental-features/delivery-timeout.md
+        - DeliverySpec.RetryAfterMax field: eventing/experimental-features/delivery-retryafter.md
+        - New Trigger Filters: eventing/experimental-features/new-trigger-filters.md
+        - KReference.Group field: eventing/experimental-features/kreference-group.md
+        - Knative reference mapping: eventing/experimental-features/kreference-mapping.md
+        - Strict Subscriber: eventing/experimental-features/strict-subscriber.md
```

Comment:
I like structuring this differently, but perhaps that stuff can be done before landing new documentation?

E.g. doing a "pre factoring" PR to do the org. stuff, and than adding the actual new doc for the trigger filters?

Reply:
I don't think it's worth it TBH. It's not a huge restructure, I just split it. Feel free to suggest any other different structure.

Reply:
No, the structure itself is good.  i was just referencing to the fact that this PR does two things:
* restructuring
* adding doc for new trigger (as the PR title)

Could be done in two PRs. But does not need to be done in two.